.idea/misc.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Didn't notice this until after submitting the pull request. I didn't edit the file directly - I used the Project Structure dialog - so I don't know why IDEA decided to remove this line.